### ISSUE TITLE
Getting node deprecation warning attempting to launch 

### ISSUE THREAD

**Issue by gumatt** [romanresh/vscode-testcafe#15]
from both context menus (in the file explore and in code) and using shortcuts the tests never execute with the following error.

```
/usr/local/bin/node --debug-brk=7668 --nolazy node_modules/testcafe/lib/cli/index.js chrome '[project root directory]/tests/endUserTests.js' --fixture 'Simple Login Test'

(node:5659) [DEP0062] DeprecationWarning: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` or `node --inspect-brk` instead.
```

Researching in the vscode repository issue  [21883](https://github.com/Microsoft/vscode/issues/21883)  suggests that launch.json should be changed; I have attempted this change and get the same results.

Is anyone else experiencing this?  Is there a work around I am not seeing?

**Comment by sijosyn** [comment]
I'm getting the same error. My node version is 8.2.1 (Mac) and VSCode ver. 1.14.2.

**Comment by romanresh** [comment]
It looks like the vscode's command "vscode.startDebug" is broken because of the new debug API which is not published yet: https://github.com/Microsoft/vscode/issues/31266 
Looks weird: the old API became broken, while the new API is not published yet (https://github.com/Microsoft/vscode/milestone/46).

I've prepared the quick fix with the experimental vscode feature (debugging with {"type": "node2"}) in v1.1.1. Try to update the extension to the new version.

After the release of vscode (with new API) the extension will be updated one more time.

**Comment by sijosyn** [comment]
@romanresh Just tried v1.1.1 and works fine. Awesome. The only issue I'm seeing right now is that the test results are not getting printed into VSCode's debug console.

**Comment by gumatt** [comment]
@romanresh I'm getting the same results.  v1.1.1 works like a champ, but test results are not displaying in debug console.

**Comment by romanresh** [comment]
Just found the cause of this problem. TestCafe uses process.stdout for test reports but VS Code has problems with this functionality in Output: https://github.com/Microsoft/vscode/issues/19750. 

I'll try to use the integrated Terminal instead of Output.

**Comment by romanresh** [comment]
Ok, the issue is fixed in v1.2.0. The extension will use Terminal for report output.

**Comment by sijosyn** [comment]
Tried v1.2.0 and it works fine. Output does get printed in Terminal. Thanks for fixing this quickly.
